python/lsst/cp/pipe/linearity.py

```diff
@@ -239,6 +239,11 @@ class LinearitySolveConfig(pipeBase.PipelineTaskConfig,
         doc="Maximum number of rejections per iteration for spline fit.",
         default=5,
     )
+    doHeaderProvenance = pexConfig.Field(
```

Comment:
Let's not make this configurable.  Adding this information is always useful.

Reply:
I was following the same thing in `isrTask`, where doHeaderProvenance` is an option [here](https://github.com/lsst/ip_isr/blob/main/python/lsst/ip/isr/isrTask.py#L399) . Should there be another ticket to remove that as well?

Reply:
No, but I acknowledge that I'm being a bit arbitrary.  Header provenance is something we may want to disable for alert production if we're not hitting the required processing times (even though the expected gain would be small).  These are being called during calib construction, when we theoretically have infinite time.